src/main/java/icons/SonarLintIcons.java

```diff
@@ -42,4 +42,8 @@ private SonarLintIcons() {
   public static Icon severity(String severity) {
     return IconLoader.getIcon("/images/severity/" + severity.toLowerCase() + ".png");
   }
+
+  public static Icon type(String type) {
+    return IconLoader.getIcon("/images/type/" + type.toLowerCase() + ".png");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Define the locale to be used in this String operation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1449)